Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Identity.Web
+{
+    public class MicrosoftIdentityOptions : OpenIdConnectOptions
```

Comment:
The is a key class that made the union of AAD and B2C helper methods possible. We need a single class that contains settings for both scenarios, in order to have it used all over Microsoft.Identity.Web, and also to check if it is a B2C app or not.